42-cellular-smoke-detector-retrofit/README.md

```diff
@@ -0,0 +1,184 @@
+
+To estimate the resistance needed to achieve a particular forward current, use Ohm's Law: V = IR. Rearrange it to get R = V/I. V is the voltage drop over your resistor. Looking at the datasheet, we see that the forward voltage, which is the voltage dropped over the LED, is typically 1.2V if the forward current is 20mA. With 6.7V coming from the source, 6.7V - 1.2V = 5.5V will be dropped over our resistor. Plugging this voltage and 20mA into R = V/I, we get R = 5.5V / 20e-3 = 275 Ω.
+
+From our testing of this particular PC817 from Gikfun ([linked above](#you-will-need)), we were able to saturate the transistor with just 0.8mA of forward current. To achieve this with a source voltage of 6.7V, we used 2 resistors in series: 1kΩ and 4.7kΩ. If you plug 5.7kΩ into the calculations above, you won't get exactly 0.8mA of forward current. In practice, however, this is what we observed when using a digital multimeter to measure the current.
+
+### Breadboarding
+
+Setting aside the Notecard and Notecarrier, which you'll assemble and configure shortly, build the circuit on the breadboard:
+
+<p align="center">
+  <img src="images/breadboard_no_notecarrier.jpg"/>
+</p>
+
+You're now ready to configure your Notehub project and the Notecard so you can run an end-to-end test of the whole system.
+
+## Notehub Setup
+
+Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#set-up-notehub). Your Notehub project is where you'll view smoke detector alarm events and where you'll configure a Twilio route to get an SMS text when an alarm happens.
+
+## Notecard Setup
+
+1. Assemble Notecard and Notecarrier as described [here](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/). Keep the Notecard connected to the your development PC with [the in-browser terminal](https://dev.blues.io/notecard-playground/) active.
+2. Set the ProductUID for the Notecard by pasting the command below into the in-browser terminal. Make sure to replace `com.your-company:your-product-name` with the [ProductUID](https://dev.blues.io/notehub/notehub-walkthrough/#finding-a-productuid) from your Notehub project.
+
+        ```json
+        { "req": "hub.set", "product": "com.your-company:your-product-name", "body": { "app": "nf42" } }
+        ```
```

Comment:
On GitHub these two code blocks render the backticks incorrectly. I think you can un-indent these to fix that.

![Screenshot 2023-08-01 at 9 44 18 AM](https://github.com/blues/app-accelerators/assets/544280/868bc637-f387-45c4-bdd5-b42de4cda39c)